lib/tools/adb-commands.js

```diff
@@ -1527,20 +1528,33 @@ methods.setHttpProxy = async function (proxyHost, proxyPort) {
   if (_.isUndefined(proxyPort)) {
     throw new Error(`Call to setHttpProxy method with undefined proxy_port ${proxy}`);
   }
-  await this.setSetting('global', 'http_proxy', proxy);
-  await this.setSetting('global', 'global_http_proxy_host', proxyHost);
-  await this.setSetting('global', 'global_http_proxy_port', proxyPort);
+
+  // Read https://android.googlesource.com/platform/frameworks/base/+/android-9.0.0_r21/core/java/android/provider/Settings.java for each property
```

Comment:
There is no need to repeat a comment twice if it was already mentioned in the docstring